app/resources/services/geoip.js

```diff
@@ -0,0 +1,65 @@
+const request = require('request')
+const redis = require('redis')
+const config = require('config')
+const logger = require('../../../lib/logger.js')()
+
+const IP_GEOLOCATION_TIMEOUT = 500
+
+exports.get = function (req, res) {
+  if (req.headers.host !== config.app_host_port) {
```

Comment:
Not sure if this is working (tried on staging: https://streetmix-staging.herokuapp.com/services/geoip) -- it shouldn't be allowed if the request isn't made by Streetmix server itself.

Reply:
Hey sorry, I think I'm confused on what you wanted. So the link itself isn't supposed to give us the information because the request needs to be made by Streetmix itself or when requesting that link in another application it worked even when it wasn't supposed to?